front/components/assistant_builder/AssistantBuilder.tsx

```diff
@@ -416,6 +434,7 @@ export default function AssistantBuilder({
                           dustApps={dustApps}
                           setBuilderState={setBuilderState}
                           setEdited={setEdited}
+                          insertAction={insertAction}
```

Comment:
could we merge insertAction and setPendingAction? so we have only one function to add to the builder?